lib/generators/decidim/app_generator.rb

```diff
@@ -73,6 +77,8 @@ def readme
       end
 
       def gemfile
+        return unless options[:gemfile]
```

Comment:
Here it is. It's `options[:skip_gemfile]`. This should be why the build is failing.

Reply:
Arghhhhhhhhhhhhhhhhhhhhhhhhhh